package.json

```diff
@@ -5,14 +5,14 @@
     "build": "rollup -c && npm run build-addon && web-ext build --overwrite-dest && mv web-ext-artifacts/*.zip web-ext-artifacts/rally_core.xpi",
     "build-addon": "rollup -c rollup.config.addon.js",
     "build-local-addon": "rollup -c rollup.config.addon.js --config-disable-remote-settings --config-study-list-url=/public/locally-available-studies.json",
-    "build-storybook": "build-storybook -s ./",
     "lint": "npm run build && npm-run-all lint-*",
     "lint-addon": "web-ext lint",
     "lint-css": "stylelint 'public/*.css' '.storybook/*.css' 'src/**/*.svelte' 'stories/**/*.svelte'",
     "lint-js": "eslint . .storybook",
     "skip-taskcluster": "node -e \"process.exit(!('TASK_ID' in process.env))\"",
     "start": "sirv public",
-    "storybook": "start-storybook -p 6006 -s ./",
+    "storybook": "start-storybook -p 6006 -s ./public",
+    "build-storybook": "build-storybook -s ./public",
```

Comment:
Nit: commands here are not sorted, would you please move it back to where it was? :P